libfreerdp/core/credssp_auth.c

```diff
@@ -104,7 +104,7 @@ rdpCredsspAuth* credssp_auth_new(const rdpContext* rdp_ctx)
 
 BOOL credssp_auth_init(rdpCredsspAuth* auth, TCHAR* pkg_name, SecPkgContext_Bindings* bindings)
 {
-	SECURITY_STATUS status;
+	SECURITY_STATUS status = ERROR_INTERNAL_ERROR;
```

Comment:
This is not needed, `status` is unconditionality set

Reply:
will change to declare where used where possible.